lib/modules/datasource/aws-eks-addon/schema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const EksAddonsFilterSchema = z.object({
+  kubernetesVersion: z.string().min(1),
+  addonName: z.string().min(1),
+  region: z.string().optional(),
+  profile: z.string().optional(),
+});
+
+export type EksAddonsFilter = z.infer<typeof EKSAddonsFilterSchema>;
+export EksAddonsFilterJson = Json.pipe(EksAddonsFilterSchema);
```

Comment:
```suggestion
export const EksAddonsFilter = Json.pipe(EksAddonsFilterSchema);
```

`Json` needs to be imported from our schema utils.